diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/SmartCastRule.kt

```diff
@@ -126,8 +126,7 @@ class SmartCastRule(private val configRules: List<RulesConfig>) : Rule("smart-ca
     @Suppress("TYPE_ALIAS")
     private fun groupIsAndAsExpr(isExpr: List<KtNameReferenceExpression>,
                                  asExpr: List<KtNameReferenceExpression>,
-                                 prop: KtProperty)
-    : Map<KtNameReferenceExpression, List<KtNameReferenceExpression>> {
+                                 prop: KtProperty): Map<KtNameReferenceExpression, List<KtNameReferenceExpression>> {
```

Comment:
```suggestion
                                 prop: KtProperty): 
    Map<KtNameReferenceExpression, List<KtNameReferenceExpression>> {
```
What if we still want to have newline? Will your rule allow such cases as above? Also, what indent will be allowed?